build.sbt

```diff
@@ -37,47 +37,63 @@ inThisBuild(List(
 
 lazy val scalatestVersion = settingKey[String]("")
 
+// shamelessly copied from cats
+def scalaVersionSpecificFolders(srcName: String, srcBaseDir: java.io.File, scalaVersion: String) = {
+  def extraDirs(suffix: String) =
+    List(CrossType.Pure, CrossType.Full)
+      .flatMap(_.sharedSrcDir(srcBaseDir, srcName).toList.map(f => file(f.getPath + suffix)))
+
+  CrossVersion.partialVersion(scalaVersion) match {
+    case Some((2, y)) if y <= 12 =>
+      extraDirs("-2.12-")
+    case Some((2, y)) if y >= 13 =>
+      extraDirs("-2.13+")
+    case _ => Nil
+  }
+}
+
 lazy val buildSettings = Seq(
   scalaVersion       := "2.12.8",
```

Comment:
shall we bump the main scala version to 2.13?

Reply:
👍